client/src/parsec/keyboard_events.ts

```diff
@@ -0,0 +1,17 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
`parsec/` is more for things that are somewhat related to parsec (bindings, devices, login, files, ...). This should be moved to `src/services` for example.